include/cocaine/rpc/asio/decoder.hpp

```diff
@@ -49,8 +51,20 @@ struct decoded_message_t {
         return object.via.array.ptr[2];
     }
 
+    boost::optional<header_t>
+    get_header(const header_key_t& key) const {
```

Comment:
sorry, I misunderstood you. I thought you're speaking about boost::optional<header_t> instead of boost::optional<const header_t>